modules/react/collection/lib/react-virtual.ts

```diff
@@ -0,0 +1,71 @@
+// retype interfaces
+// delete when `react-virtual` accepts this PR: https://github.com/TanStack/react-virtual/pull/277
```

Comment:
Can be removed when https://github.com/TanStack/react-virtual/pull/277 is merged